src/openforms/contrib/objects_api/tests/factories.py

```diff
@@ -7,6 +9,7 @@
 
 class ObjectsAPIGroupConfigFactory(factory.django.DjangoModelFactory):
     name = factory.Sequence(lambda n: f"Objects API group {n:03}")
+    identifier = factory.LazyAttribute(lambda o: slugify(o.name))
```

Comment:
even more simple and less likely to result in unique constraint violations (consider the case `ObjectsAPIGroupConfigFactory.create_batch(2, name="foo")` -> both will end up with the same slug

```suggestion
    identifier = factory.Sequence(lambda n: f"objects-api-group-{n}")
```

the factory boy sequence is tracked globally and will never emit the same sequence number -> guaranteed unique slug.